<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CookiesExceptionsWidget</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="CookiesExceptionWidget">
     <property name="title">
      <string>New Exception</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Domain:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="domainLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="_3">
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>81</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="blockButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Block</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="allowForSessionButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Allow For Session</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="allowButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Allow</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ExceptionsGroupBox">
     <property name="title">
      <string>Exceptions</string>
     </property>
     <layout class="QGridLayout" name="_4">
      <item row="1" column="0" colspan="5">
       <widget class="EditTableView" name="exceptionTable"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>&amp;Remove</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="removeAllButton">
        <property name="text">
         <string>Remove &amp;All</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="KLineEdit" name="search"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>EditTableView</class>
   <extends>QTableView</extends>
   <header>edittableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
